src/sys/reboot.rs

```diff
@@ -0,0 +1,50 @@
+//! Reboot/shutdown or enable/disable Ctrl-Alt-Delete.
+
+use {Errno, Error, Result};
+use libc;
+use void::Void;
+use std::mem::drop;
+
+/// How exactly should the system be rebooted.
+///
+/// See [`set_cad_enabled()`](fn.set_cad_enabled.html) for
+/// enabling/disabling Ctrl-Alt-Delete.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum RebootMode {
```

Comment:
So maybe it is a bug in GH/Travis. Let me change the names and try `#[repr(i32)]` one more time then.